catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -111,6 +112,15 @@ public UpdateRequest process(UpdateRequest input)
                 boolean validationErrorsExist = CollectionUtils.isNotEmpty(e.getErrors());
                 boolean validationWarningsExist = CollectionUtils.isNotEmpty(e.getWarnings());
 
+                ValidationResult validationResult = new ValidationResult(false, false);
```

Comment:
i think you could just say `ValidationResult validationResult = new ValidationResult(validationErrorsExist, validationWarningsExist)` and save a few lines